app/scripts/modules/core/src/managed/constraints/constraintRegistry.tsx

```diff
@@ -61,5 +69,17 @@ export const statefulConstraintOptionsByType: { [type: string]: IStatefulConstra
           return throwUnhandledStatusError(status);
       }
     },
+    overrideActions: {
+      [PENDING]: [
+        {
+          title: 'Reject',
+          pass: false,
+        },
+        {
+          title: 'Approve',
+          pass: true,
+        },
+      ],
+    },
```

Comment:
i did _not_ want to enable anyone sending up arbitrary statuses to persist on constraints — the whole point of `OVERRIDE_PASS` and `OVERRIDE_FAIL` are to be a way of understanding that a human or other thing external to the constraint's own evaluator decided to 'demand' the a particular status was applied.